rocksdb_replicator/rocksdb_replicator.h

```diff
@@ -127,7 +128,7 @@ class RocksDBReplicator {
     std::shared_ptr<rocksdb::DB> db_;
     folly::Executor* const executor_;
     const DBRole role_;
-    const folly::SocketAddress upstream_addr_;
+    folly::SocketAddress upstream_addr_;
```

Comment:
I don't fully remember the threading model for this code. Will take a closer look at it.

Can you please think through its threading model and the interaction between replicatedDB, replicator and db manager in case you haven't yet.

Reply:
Yes, will look into it.

Reply:
@newpoo Didn't find any issues with removing the const here. Application DB stores a copy of the upstream, and it will be out of sync with the upstream here, but that's not used once it is set, so it should be fine. Even within the replicator, once the upstream is set, it is not used for anything but sending replication requests, so should be fine too. Please lmk what you think